extensions/ql-vscode/package.json

```diff
@@ -361,6 +361,10 @@
         "command": "codeQL.quickQuery",
         "title": "CodeQL: Quick Query"
       },
+      {
+        "command": "codeQL.createSkeletonQuery",
+        "title": "CodeQL: Create Query"
```

Comment:
Should this command immediately be available to all users or should this be feature flagged?

Reply:
I've discussed this with our PM and we want this to be available.

Reply:
Added here: https://github.com/github/vscode-codeql/pull/2305